lib/Basic/Platform.cpp

```diff
@@ -23,48 +23,47 @@ bool swift::tripleIsiOSSimulator(const llvm::Triple &triple) {
   return (triple.isiOS() &&
           // FIXME: transitional, this should eventually stop testing arch, and
           // switch to only checking the -environment field.
-          (triple.isSimulatorEnvironment() ||
-           arch == llvm::Triple::x86 || arch == llvm::Triple::x86_64));
+          (triple.isSimulatorEnvironment() || arch == llvm::Triple::x86 ||
```

Comment:
These line break changes here are unrelated to the commit; please clang-format only your own changes and restore this to the original version.

Reply:
This is not resolved. Please restore the original whitespace.